debian/autobake-deb.sh

```diff
@@ -111,9 +119,10 @@ in
   "buster")
     disable_libfmt
     replace_uring_with_aio
-    ;&
+    ;;
```

Comment:
probably safer to remove "buster" (EOL) as all the previous following cases are now removed because of `;;`.